tests/test-helper.ts

```diff
@@ -2,7 +2,7 @@
 //@ts-nocheck
 // TODO: linting disabled cause of really bad and incomplete qunit typings, fix
 import * as QUnit from 'qunit';
-import Application from 'dummy/app';
+import Application from 'dummy/tests/dummy/app/app';
```

Comment:
Why did you include this change. My editor and tsc complains because of this change. It works well with `dummy/app`

Reply:
probably some fat-fingered vim commands if I'd have to guess

Reply:
I am curious tho why your tsc complained and mine didn't, especially considering this is in a ts-nocheck file

Reply:
My bad, it wasn't ts complaining, it was a runtime error